src/Paket.Core/Paket.Core/project.json

```diff
@@ -0,0 +1,124 @@
+{
+  "version": "3.15.4",
+  "buildOptions": {
+    "debugType": "portable",
+    "compilerName": "fsc",
+    "define": [
+      "DOTNETCORE"
+    ],
+    "emitEntryPoint": false,
+    "xmlDoc": true,
+    "compile": {
+      "includeFiles": [
+        "../../../paket-files/fsprojects/FSharp.TypeProviders.StarterPack/src/AssemblyReader.fs",
+        "../../../paket-files/fsharp/FAKE/src/app/FakeLib/Globbing/Globbing.fs",
+        "../Async.fs",
+        "../AssemblyInfo.fs",
+        "../CustomAssemblyInfo.fs",
+        "../Domain.fs",
+        "../Logging.fsi",
+        "../Logging.fs",
+        "../Constants.fs",
+        "../Utils.fs",
+        "../SemVer.fs",
+        "../VersionRange.fs",
+        "../Xml.fs",
+        "../GitCommandHelper.fs",
+        "../GitHandling.fs",
+        "../PlatformDetection.fs",
+        "../ConfigFile.fs",
+        "../Cache.fs",
+        "../Cultures.fs",
+        "../PackageSources.fs",
+        "../FrameworkHandling.fs",
+        "../PlatformMatching.fs",
+        "../Requirements.fs",
+        "../ModuleResolver.fs",
+        "../RemoteDownload.fs",
+        "../RemoteUpload.fs",
+        "../PackageResolver.fs",
+        "../Nuspec.fs",
+        "../InstallModel.fs",
+        "../ReferencesFile.fs",
+        "../SolutionFile.fs",
+        "../Nuget.fs",
+        "../NuGetV3.fs",
+        "../NuGetV2.fs",
+        "../DependenciesTypes.fs",
+        "../DependenciesFileParser.fs",
+        "../LockFile.fs",
+        "../TemplateFile.fs",
+        "../ProjectFile.fs",
+        "../DependenciesFile.fs",
+        "../LocalFile.fs",
+        "../DependencyChangeDetection.fs",
+        "../GarbageCollection.fs",
+        "../RestoreProcess.fs",
+        "../BindingRedirects.fs",
+        "../NupkgWriter.fs",
+        "../ProcessOptions.fs",
+        "../PackagesConfigFile.fs",
+        "../DependencyModel.fs",
+        "../InstallProcess.fs",
+        "../UpdateProcess.fs",
+        "../RemoveProcess.fs",
+        "../AddProcess.fs",
+        "../PackageMetaData.fs",
+        "../PackageProcess.fs",
+        "../Environment.fs",
+        "../Releases.fs",
+        "../Simplifier.fs",
+        "../VSIntegration.fs",
+        "../NugetConvert.fs",
+        "../FindOutdated.fs",
+        "../FindReferences.fs",
+        "../PublicAPI.fs",
+        "../ScriptGeneration.fs"
```

Comment:
https://github.com/fsprojects/Paket/pull/1785/files#diff-6074983490330a0aac1bd23c958232d7R151 We should make that a FAKE thing :D